build.sbt

```diff
@@ -48,12 +48,7 @@ lazy val examples = Project("blaze-examples",
 val dontPublish = packagedArtifacts := Map.empty
 
 val jvmTarget = TaskKey[String]("jvm-target-version", "Defines the target JVM version for object files.")
-jvmTarget in ThisBuild := {
-  VersionNumber(scalaVersion.value).numbers match {
-    case Seq(2, 10, _*) => "1.7"
-    case _ => "1.8"
-  }
-}
+jvmTarget in ThisBuild := "1.8"
```

Comment:
Looks like this was actually necessary because 2.10 doesn't want to emit java 8 bytecode.